package.json

```diff
@@ -86,9 +90,17 @@
     "webpack-hot-middleware": "^2.12.2"
   },
   "jest": {
-    "testRegex": "(/test/.*\\.spec.js)$",
+    "testRegex": "(/test/unit/.*\\.spec.js)$",
     "setupFiles": [
-      "test/env.js"
+      "test/unit/env.js"
+    ],
+    "collectCoverageFrom": [
```

Comment:
Is there any way we can work around this? For example in Python you can instruct the coverage tool to "discover" all existing files in the project and include them in the report, even if they are not covered. This is an easy shortcut to say "print me a coverage report for everything". 

OTOH in Haskell we can't autodiscover files and include them in coverage. I'd like to avoid this for JavaScript if possible. The problem is that once you whitelist some files it is very easy to get a high number of coverage for them and leave it as is. Then when a new directory is added the new files will not show in the report and it will still remain at the previous number.  Which may lead to the false impression that there is sufficient test coverage while there isn't.  Can we avoid that somehow? Maybe try a different coverage collection tool for JS if available.